data/themes/darktable-elegant-dark.css

```diff
@@ -72,10 +72,6 @@
 @define-color field_hover_bg @grey_60;
 @define-color field_hover_fg @grey_20;
 
-/* Tooltips and contextual helpers */
```

Comment:
I'm not sure it was a good idea to remove this after all. The tooltip black backgroud for a grey or dark theme is a bit disturbing I found. What was the rational for this change?

Reply:
That's why I put it back on grey theme. I could also see that on my #3362 PR if you want. I will note that also for dark theme.

Reply:
right click on a slider with this change and you will see that the cursor border (triangle) has a different colour than the popup. This is because I haven't been able to dynamically change the sliders border color. I agree it's not consistent but it's less ugly until I find a way to change cursor border color with CSS rules.